examples/with-typescript/tsconfig.json

```
@@ -0,0 +1,27 @@
+{
+  "compilerOptions": {
+    "target": "esnext",
```

Target should be now `es2018` (or `es2019`/`es2020`) to support last TypeScript features, see note in [**Nuxt TypeScript documentation**](https://typescript.nuxtjs.org/guide/setup.html#configuration).

```suggestion
    "target": "es2018",
```

---

Sounds good.